xds/src/test/java/io/grpc/xds/XdsSdsClientServerTest.java

```diff
@@ -82,12 +81,11 @@
   @Rule public final GrpcCleanupRule cleanupRule = new GrpcCleanupRule();
   private int port;
   private FakeNameResolverFactory fakeNameResolverFactory;
-  private Bootstrapper mockBootstrapper;
+  private final TlsContextManagerImpl tlsContextManager = new TlsContextManagerImpl(null);
```

Comment:
`new TlsContextManagerImpl(null)` passes in a `null` BootstrapInfo, but not a single line of code in ClientSslContextProviderFactory/ServerSslContextProviderFactory does any null check and runs with dedicated codepath. So I am really questioning the tests...